rbac/management/group/relation_api_dual_write_group_handler.py

```diff
@@ -113,9 +113,7 @@ def _replicate(self):
                 ReplicationEvent(
                     event_type=self.event_type,
                     info={"group_uuid": str(self.group.uuid)},
-                    # TODO: need to think about partitioning
-                    # Maybe resource id
-                    partition_key="rbactodo",
+                    partition_key="settings.ENV_NAME",
```

Comment:
@wcmitchell do you mean to have here value of settings.ENV_NAME ? If so, it will be without quotes ` partition_key=settings.ENV_NAME` or [example](https://github.com/RedHatInsights/insights-rbac/blob/93eebbf2d4c9d5119a14e6eed8b81fc7d8e866bf/rbac/management/tenant_service/v2.py#L279)